prettier.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
     arrowParens: 'avoid',
     bracketSpacing: true,
-    endOfLine: 'crlf',
```

Comment:
line ending changes scare me after hitting issues with them in the docs generator repo, why was this necessary?

Reply:
It is same problem we had with the samples repo, depending OS the end of line is different. In this case PR was not passing the lint checks as end of line was detected different (it runs on Linux).
Found that flipping this setting avoids this problem.

Reply:
Ah, alright. To align these settings with the docs generator and minecraft, you could keep this as crlf and add a `.gitattributes` file with `* text eol=crlf`. Then prettier should behave the same as our other ones.

Reply:
Ok, changing to this mechanism